BlockEQ/View Controllers/Wallet/AddressEntryViewController.swift

```diff
@@ -70,8 +70,26 @@ final class AddressEntryViewController: UIViewController {
         let availableSendBalance = account.availableSendBalance(for: asset).displayFormattedString
         navigationItem.title = String(format: "TRADE_BALANCE_FORMAT".localized(), availableSendBalance, asset.shortCode)
     }
+    
+    @IBAction func textFieldEditingDidChange(_ sender: Any) {
+        let text = sendAddressTextField.text!
```

Comment:
I like to explicitly avoid fore-unwrapping values here, whenever a nil value is unwrapped, the app will force crash. 

Usually force unwrapping is used when you need to tell the compiler that you are absolutely sure the value you're about to use will *never* be nil, like if you were to do the following:

```swift
if textField.text != nil { 
   print("I'm fore sure not nil: \(textField.text!)")
}
```

But Swift gives nicer tools to handle things like this, such as:

```swift
guard let text = textField.text else { return } 

// Now we either return from this method if there's no text, or everything 
// below this line has a locally scoped, non-optional `text` constant you can use.